api/v1alpha1/validation_test.go

```diff
@@ -62,17 +62,16 @@ func TestEtcdCluster_ValidateUpdate(t *testing.T) {
 		err      string
 	}{
 		{
-			name: "ScaleOut",
+			name: "ScaleUp",
 			modifier: func(o *v1alpha1.EtcdCluster) {
 				*o.Spec.Replicas += 1
 			},
 		},
 		{
-			name: "UnsupportedChange/ScaleIn",
+			name: "ScaleDown",
```

Comment:
Using the term Scale Down rather than Scale In, as discussed with @JamesLaverack ....although I believe scale-in and scale-out are the correct terms.